cmd/upgrade/main.go

```diff
@@ -17,33 +17,28 @@ limitations under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
 
-	log "github.com/golang/glog"
-	upgrade "github.com/openebs/maya/cmd/upgrade/app/v1alpha1"
+	upgrade090to100 "github.com/openebs/maya/pkg/upgrade/v1alpha2/0.9.0-1.0.0"
 )
 
 func main() {
-	err := flag.Set("logtostderr", "true")
-	if err != nil {
-		fmt.Printf("failed to upgrade: %s", err)
-		os.Exit(1)
-	}
-	configPath := flag.String("config-path", "/etc/config/upgrade", "path to upgrade config file.")
-	defer log.Flush()
-	flag.Parse()
-
-	u, err := upgrade.NewUpgradeForConfigPath(*configPath)
-	if err != nil {
-		log.Errorf("failed to upgrade: %+v", err)
-		os.Exit(1)
-	}
-
-	err = u.Run()
-	if err != nil {
-		log.Errorf("failed to upgrade: %+v", err)
+	from := os.Args[1]
+	to := os.Args[2]
+	kind := os.Args[3]
+	name := os.Args[4]
+	openebsNamespace := os.Args[3]
```

Comment:
Hmm.. Args 3 are both kind and openebsNamespace? How about changing this into CORBA CLI based argument parsing.. so we don't have to depend on the positional parsing. Either in this PR or can be taken up in the subsequent PR.

Reply:
Done